### DISCUSSION TITLE
Meaning of W_fuel

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13915] [Q&A] [answered]
In the Thermal degradation model, section 17.1.2, a quantity W_fuel (Eq. 17.24) is defined. It seems to represent the molecular weight of something, but I'm not sure which component. At first I thought it was the molecular weight of the fuel gas, however, would that not simply be 12(x−x′) +y+16(z−z′)? Could anyone clarify?

**Comment by mcgratta** [comment]
I'll take a look.

**Comment by mcgratta** [comment, marked as answer]
Yes, that seems confusing. W_fuel in this context is referring to the original dry vegetation. It is assumed that the dry vegetation is made up of a single effective molecule, whose molecular weight is W_fuel. I'll change the subscript to "veg".

**Reply by r-broek** [reply]
Ah that makes sense. As a follow-up question: does the $1-\nu_{ash} \nu_{char}$ term then account for the molecular weight of the inorganic compounds $A$?

**Reply by mcgratta** [reply]
Yes, and $\nu_{\rm ash}$ is the fraction of char mass that remains as ash.

**Reply by r-broek** [reply]
I think I understand it now, thanks a lot!
